homeassistant/components/konnected/sensor.py

```diff
@@ -57,7 +70,7 @@ def async_add_ds18b20(attrs):
                 KonnectedSensor(
                     device_id,
                     sensor_config,
-                    DEVICE_CLASS_TEMPERATURE,
+                    SENSOR_TYPES[0],
```

Comment:
There are no tests for this integration to verify this.
When a new sensor is added as the first entry in `SENSOR_TYPES` this will be wrong.

SENSOR_TYPES should remain a dict, so we can do lookup here for the temperature type to prevent issues in the future.